docs/design/dynamic-mig.md

```diff
@@ -0,0 +1,158 @@
+        allowedGeometries:
+          - 
+            - name: 1g.5gb
+              memory: 5120
+              count: 7
+          - 
+            - name: 2g.10gb
+              memory: 10240
+              count: 3
+            - name: 1g.5gb
+              memory: 5120
+              count: 1
+          - 
+            - name: 3g.20gb
+              memory: 20480
+              count: 2
+          - 
+            - name: 7g.40gb
+              memory: 40960
+              count: 1
+      - models: [ "A100-SXM4-80GB", "A100-80GB-PCIe", "A100-PCIE-80GB"]
+        allowedGeometries:
+          - 
+            - name: 1g.10gb
+              memory: 10240
+              count: 7
+          - 
+            - name: 2g.20gb
+              memory: 20480
+              count: 3
+            - name: 1g.10gb
+              memory: 10240
+              count: 1
+          - 
+            - name: 3g.40gb
+              memory: 40960
+              count: 2
+          - 
+            - name: 7g.79gb
+              memory: 80896
+              count: 1
+      nodeconfig: 
+          - name: nodeA
+            operatingmode: hami-core
+          - name: nodeB
+            operatingmode: mig
+```
+
+## Structure
+
+<img src="./images/volcano-dynamic-mig-structure.png" width = "400" /> 
+
+## Examples
+
+Dynamic mig is compatable with volcano-vgpu tasks, as the example below: 
+Just Setting `volcano.sh/vgpu-number` and `volcano.sh/vgpu-memory`.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: gpu-pod1
+spec:
+  containers:
+    - name: ubuntu-container1
+      image: ubuntu:20.04
+      command: ["bash", "-c", "sleep 86400"]
+      resources:
+        limits:
+          volcano.sh/vgpu-number: 2 # requesting 2 vGPUs
+          volcano.sh/vgpu-memory: 8000 # Each vGPU contains 8000m device memory （Optional,Integer
+```
+
+A task can decide only to use `mig` or `hami-core` by setting `annotations.volcano.sh/vgpu-mode` to corresponding value, as the example below shows:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: gpu-pod1
+  annotations:
+    volcano.sh/vgpu-mode: "mig"
+spec:
+  containers:
+    - name: ubuntu-container1
+      image: ubuntu:20.04
+      command: ["bash", "-c", "sleep 86400"]
+      resources:
+        limits:
+          volcano.sh/vgpu-number: 2 # requesting 2 vGPUs
+          volcano.sh/vgpu-memory: 8000 # Each vGPU contains 8000m device memory （Optional,Integer）
+```
+
+## Procedures
+
+The Procedure of a vGPU task which uses dynamic-mig is shown below:
+
+<img src="./images/volcano-dynamic-mig-procedure.png" width = "800" /> 
+
+Note that after submited a task, deviceshare plugin will iterate over templates defined in configMap `volcano-device-share`, and find the first available template to fit. You can always change the content of that configMap, and restart vc-scheduler to customize.
+
+If you submit the example on an empty A100-PCIE-40GB node, then it will select a GPU and chosse MIG template below:
```

Comment:
If you submit a pod requiring 4GPU, 20GB device memory, and an empty A100-PCIE-40GB node has the following MIG template below: 
```yaml
  2g.10gb: 3
  1g.5gb: 1
```
Then two `2g.10gb` profiles will be allocated to this pod, should be like this? Your example here doesn't say how much resources the Pod requests.

Reply:
typo `chosse`